ts/screens/wallet/payment/ConfirmPaymentMethodScreen.tsx

```diff
@@ -89,8 +91,11 @@ export type ConfirmPaymentMethodScreenNavigationParams = Readonly<{
   psps: ReadonlyArray<Psp>;
 }>;
 
-type OwnProps =
-  NavigationStackScreenProps<ConfirmPaymentMethodScreenNavigationParams>;
+type OwnProps = {
+  navigation: CompatNavigationProp<
+    IOStackNavigationProp<WalletParamsList, "PAYMENT_CONFIRM_PAYMENT_METHOD">
```

Comment:
Since the `navigation` in the props and the `useNavigation` hook must be typed as the same, what do you think about moving `IOStackNavigationProp<WalletParamsList, "PAYMENT_CONFIRM_PAYMENT_METHOD">` to a separate type alias?

Reply:
Better for sure! Changed in 98b011359cd56d86ee36bbc272bbd415c96d1672